Sources/Amplitude/Amplitude.m

```diff
@@ -114,6 +116,12 @@ @interface Amplitude ()
 static NSString *const USER_ID = @"user_id";
 static NSString *const SEQUENCE_NUMBER = @"sequence_number";
 
+static NSString *const OLD_USER_ID = @"user_id";
```

Comment:
OLD and NEW have the same key?

Reply:
I believe in new storage @yuhao900914 is trying to use `getDataStorageKey` for new key, but I guess we don't need to define the old const for it.

Reply:
I see, makes sense.

Reply:
Yes, the key for new storage is using `getDataStorageKey`, change to OLD_XXX is for making it more clear those keys are points to the old key for database.